mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -694,22 +695,24 @@ def _get_v3io_frames_client(
         )
 
     @staticmethod
-    def _get_endpoint_filter(endpoint_id: Union[str, list[str]]) -> Optional[str]:
-        if isinstance(endpoint_id, str):
-            return f"endpoint_id=='{endpoint_id}'"
-        elif isinstance(endpoint_id, list):
-            if len(endpoint_id) > V3IO_FRAMESD_MEPS_LIMIT:
+    def _generate_filter_query(
+        filter_key: str, filter_values: Union[str, list[str]]
+    ) -> Optional[str]:
+        if isinstance(filter_values, str):
+            return f"{filter_key}=='{filter_values}'"
+        elif isinstance(filter_values, list):
+            if len(filter_values) > V3IO_FRAMESD_MEPS_LIMIT:
                 logger.info(
-                    "The number of endpoint ids exceeds the v3io-engine filter-expression limit, "
-                    "retrieving all the model endpoints from the db.",
+                    "The number of filter values exceeds the v3io-engine filter-expression limit, "
+                    "retrieving all the values from the db.",
                     limit=V3IO_FRAMESD_MEPS_LIMIT,
```

Comment:
We will probably want to know what field this is
```suggestion
                    "The number of filter values exceeds the v3io-engine filter-expression limit, "
                    "retrieving all the values from the db.",
                    filter_key=filter_key,
                    limit=V3IO_FRAMESD_MEPS_LIMIT,
```